jscomp/build_tests/super_errors/expected/modules1.res.expected

```diff
@@ -7,7 +7,5 @@
 
   [1;33mThe module or file Foo can't be found.[0m
   - If it's a third-party dependency:
-    - Did you list it in bsconfig.json?
-    - Did you run `rescript build` instead of `rescript build -with-deps`
-      (latter builds third-parties)?
-  - Did you include the file's directory in bsconfig.json?
\ No newline at end of file
+    - Did you add it to the "bs-dependencies" or "bs-dev-dependencies" in bsconfig.json?
+  - Did you include the file's directory to the "sources" in bsconfig.json?
```

Comment:
Highlighting that I've changed the text a little bit.

Reply:
I am indifferent. What do you think @cknitt @cristianoc @zth ?

Reply:
I think it's fine to leave it like it is. We'll do a search replace for bsconfig -> rescript anyway when it's time.